platform/core.netigso/test/unit/src/org/netbeans/core/netigso/NetigsoOSGiActivationVisibleTest.java

```diff
@@ -203,19 +203,19 @@ public void testResourcesFromModule() throws Exception {
 
     public void testResourcesDirectFromBundle() throws Exception {
         Method loadResource = directBundle.getMethod("loadResources", String.class, ClassLoader.class);
-        Enumeration res = (Enumeration) loadResource.invoke(null, "org/foo/Something.txt", null);
+        Enumeration<URL> res = (Enumeration<URL>) loadResource.invoke(null, "org/foo/Something.txt", null);
```

Comment:
This results in unchecked cast warnng, right?

Reply:
True.
The aim of this PR was too get the number of rawtypes warnings concerning Enumeration reduced. In this project this resulted in fixing the rawtypes warning by adding an unchecked cast warning.
This is also one of the reasons why I switched to fix as many (hopefully all) warnings in a single project.